libmamba/CMakeLists.txt

```diff
@@ -162,8 +162,8 @@ set(
     ${LIBMAMBA_SOURCE_DIR}/util/path_manip.cpp
     ${LIBMAMBA_SOURCE_DIR}/util/random.cpp
     ${LIBMAMBA_SOURCE_DIR}/util/string.cpp
-    ${LIBMAMBA_SOURCE_DIR}/util/url.cpp
     ${LIBMAMBA_SOURCE_DIR}/util/url_manip.cpp
+    ${LIBMAMBA_SOURCE_DIR}/util/url.cpp
```

Comment:
This is the correct sort order, and this is how it shows up in the tree list in VS Code, for example